cmd/util/ledger/migrations/storage_v4.go

```diff
@@ -242,6 +242,16 @@ var tokenForwardingLocationMainnet = common.AddressLocation{
 	Name:    "TokenForwarding",
 }
 
+var tokenHolderKeyManagerLocationMainnet1 = common.AddressLocation{
+	Address: common.BytesToAddress([]byte{0x93, 0x7c, 0xbd, 0xee, 0x13, 0x5c, 0x65, 0x6c}),
+	Name:    "TokenHolderKeyManager",
+}
+
+var tokenHolderKeyManagerLocationMainnet2 = common.AddressLocation{
+	Address: common.BytesToAddress([]byte{0x7f, 0x56, 0x0d, 0x7e, 0x3f, 0xf0, 0x4a, 0x31}),
+	Name:    "TokenHolderKeyManager",
+}
+
```

Comment:
Hey @psiemens could you please double-check these as well.

Reply:
These are correct 👍